README.md

```diff
@@ -68,6 +68,7 @@ gnag {
         severity 'Error'
     }
     github {
+        rootUrl 'https://my.githubinstall.com/repos/'
```

Comment:
This should be `api.my.githubinstall.com/repos` or equivalent. Basically the API authority + host of the Github instance.